R/utils.R

```diff
@@ -76,7 +76,7 @@ file_url <- function(filename) {
   anchor_suffix <- sub(file, "", filename)
   enc2(paste0(
     "file:///",
-    normalizePath(path = file, mustWork = TRUE),
+    normalizePath(path = file, mustWork = TRUE, winslash = "/"),
```

Comment:
This is still useful to do IMO on windows, but it won't be a problem for the issue I think as Windows handles the conversion. 

Probably worth testing... Ping me when PR is updated, and I'll try the PR of webshot2 with rgl on windows